### DISCUSSION TITLE
Integration with Nostr

### DISCUSSION THREAD

**Discussion by rafael-xmr** [dailydotdev/daily#850] [Feature Requests]
Ability to connect an account with a nostr-enabled signing extension & use your profile for commenting, sharing & relay your activity over the nostr network where anyone in your contact list can participate in the discussion from their favorite clients https://nostr.how/

edit: also Dev Cards could be stored as nostr badges https://badges.page

**Comment by idoshamun** [comment]
Hi @saltrafael,

I'm not familiar with nostr. Thanks for the suggestion, we'll check it out.

**Reply by rafael-xmr** [reply]
this is what a profile looks like https://snort.social/p/seth@sethforprivacy.com, it may look different on other clients depending on features implemented, like: https://iris.to/seth@sethforprivacy.com, but the idea is simple, no crypto or web3 involved if it's not clear, only [notes and other stuff transmitted by relays](https://github.com/nostr-protocol/nostr), filled with people passionate about code & open source, mainly it'd benefit a lot from the companion extension and being able to share or comment straight from articles and get instant feedback/reactions and drive discussions on the network
